packages/tux/src/components/fields/DatePicker.tsx

```diff
@@ -95,12 +96,14 @@ class DatePicker extends Component<any, State> {
           onFocus={this.handleInputFocus}
           onBlur={this.handleInputBlur}>
           <input
+            id="dateInput"
```

Comment:
Editor components get an id which they should use, so they work with the label, which is rendered uniformly.